lab.R

```diff
@@ -166,29 +192,42 @@ select(dataf, Sale_Price, Overall_Cond, Gr_Liv_Area)
 #' We will use `where()`, which allows us to use a predicate function (function that returns `TRUE` or `FALSE`) to select all of the columns that are factors or numerical. 
 #' 
 #' Fill in the blanks in the following code.  Hints: Use `?factor` and `?numeric` to bring up documentation for these classes.  Look for functions that start with `is`.  Check the documentation for `where()` to see examples. 
-
-# dataf_smol = select(???, where(???), where(???))
+?factor
+?numeric
+dataf_smol = select(dataf, where(is.numeric), where(is.factor))
+rlang::last_error()
```

Comment:
I'm guessing you got an error message that suggested this.  It's a debugging function, used to track down the causes of errors.  You shouldn't need it in runtime code. 
```suggestion
```